test/fuzz/transformation_access_chain_test.cpp

```diff
@@ -184,14 +184,16 @@ TEST(TransformationAccessChainTest, BasicTest) {
                    .IsApplicable(context.get(), transformation_context));
 
   // Bad: pointer is null
-  ASSERT_FALSE(TransformationAccessChain(
+  ASSERT_DEATH(TransformationAccessChain(
                    100, 45, {80}, MakeInstructionDescriptor(24, SpvOpLoad, 0))
```

Comment:
Uses of ASSERT_DEATH need to be wrapped in:

```
#ifndef NDEBUG
// comment
ASSERT_DEATH(...);
#endif
```

See transformation_permute_phi_operands_test.cpp for an example.

This is to keep release compilers happy - death tests don't work in release mode.

Reply:
Done